community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_3/CostCompatibility.scala

```diff
@@ -62,7 +62,7 @@ case class CostCompatibility[CONTEXT <: CommunityRuntimeContext,
   override val maybeRuntimeName: Option[RuntimeName] = runtime match {
     case CypherRuntime.default => None
     case CypherRuntime.interpreted => Some(InterpretedRuntimeName)
-    case CypherRuntime.registerInterpreted => Some(EnterpriseInterpretedRuntimeName)
+    case CypherRuntime.`slotted` => Some(SlottedRuntimeName)
```

Comment:
I believe intellij did that for me. When I saw it, I assumed there was a valid reason, but did not investigate what that was.